bitnami/cert-manager/values.yaml

```diff
@@ -0,0 +1,939 @@
+  ##
+  extraEnvVars: []
+
+  ## ConfigMap with extra environment variables
+  ## @param cainjector.extraEnvVarsCM Name of existing ConfigMap containing extra env vars
+  ##
+  extraEnvVarsCM:
+
+  ## Secret with extra environment variables
+  ## @param cainjector.extraEnvVarsSecret Name of existing Secret containing extra env vars
+  ##
+  extraEnvVarsSecret:
+
+  ## Extra volumes to add to the deployment
+  ## @param cainjector.extraVolumes  Optionally specify extra list of additional volumes for CAInjector pods
+  ##
+  extraVolumes: []
+
+  ## Extra volume mounts to add to the container
+  ## @param cainjector.extraVolumeMounts  Optionally specify extra list of additional volumeMounts for CAInjector container(s)
+  ##
+  extraVolumeMounts: []
+
+  ## Add init containers to the cainjector pods.
+  ## Example:
+  ## initContainers:
+  ##   - name: your-image-name
+  ##     image: your-image
+  ##     imagePullPolicy: Always
+  ##     ports:
+  ##       - name: portname
+  ##         containerPort: 1234
+  ## @param cainjector.initContainers  Add additional init containers to the CAInjector pods
+  ##
+  initContainers: {}
+
+  ## Add sidecars to the cainjector pods.
+  ## Example:
+  ## sidecars:
+  ##   - name: your-image-name
+  ##     image: your-image
+  ##     imagePullPolicy: Always
+  ##     ports:
+  ##       - name: portname
+  ##         containerPort: 1234
+  ## @param cainjector.sidecars  Add additional sidecar containers to the CAInjector pod
+  ##
+  sidecars: {}
+
+  ## ServiceAccount configuration
+  ##
+  serviceAccount:
+    ## @param cainjector.serviceAccount.create Specifies whether a ServiceAccount should be created
+    ##
+    create: true
+    ## @param cainjector.serviceAccount.name The name of the ServiceAccount to use.
+    ## If not set and create is true, a name is generated using the common.names.fullname template
+    ##
+    name: ""
+    ## @param cainjector.serviceAccount.annotations Additional custom annotations for the ServiceAccount
+    ##
+    annotations: {}
+
+## @section Metrics Parameters
+metrics:
+  ## @param metrics.enabled Start a sidecar prometheus exporter
+  ##
+  enabled: true
+  ## Prometheus Service Monitor
+  ## ref: https://github.com/coreos/prometheus-operator
+  ##      https://github.com/coreos/prometheus-operator/blob/master/Documentation/api.md#endpoint
+  ##
+
+  ## @param metrics.podAnnotations [object] Annotations for Cert Manager exporter pods
+  ## ref: https://kubernetes.io/docs/concepts/overview/working-with-objects/annotations/
+  ##
+  podAnnotations:
+    prometheus.io/path: "/metrics"
+    prometheus.io/scrape: "true"
+    prometheus.io/port: "9402"
+
+  serviceMonitor:
+    ## @param metrics.serviceMonitor.enabled Create ServiceMonitor resource(s) for scraping metrics using PrometheusOperator
+    ##
+    enabled: false
+    ## @param metrics.serviceMonitor.targetPort The port in which the ServiceMonitor will monitor
+    ##
+    targetPort: 9402
```

Comment:
Remove this parameter, you can dynamically set it in the **servicemonitor.yaml**
```
```